definitions/procedures/puppet/enable_puppet.rb

```diff
@@ -0,0 +1,49 @@
+module Procedures::Puppet
+  class EnablePuppet < ForemanMaintain::Procedure
+    metadata do
+      description 'Enable Puppet feature'
+      param :without_ca, 'Do not enable Puppet CA'
+      advanced_run false
+    end
+
+    def run
+      services = feature(:foreman_server).services + feature(:dynflow_sidekiq).services
+      Procedures::Service::Stop.new(:only => services)
```

Comment:
Stopping services isn't needed on installation, I'd feel safe to relying on the installer to do the right thing.

Reply:
Enable is no more :)